ui/src/audioplayer/Player.js

```diff
@@ -93,6 +95,7 @@ const Player = () => {
     () => ({
       theme: playerTheme,
       bounds: 'body',
+      playMode: localStorage.getItem(PLAYMODE_KEY),
```

Comment:
Ideally we would store this info in the Redux store, and add the selector to the store persistence, in https://github.com/navidrome/navidrome/blob/25d6b8d4956aa53bc53afdacfc9540e73a038355/ui/src/store/createAdminStore.js#L58-L67

This would be the ideal implementation in the current UI. But, as we are moving away from both Redux and this player in the next UI, and the current one is basica legacy at this point, I'm ok with this implementation. Unless you want to implement it using Redux :P